chsdi/templates/htmlpopup/sug_planning.mako

```diff
@@ -17,6 +18,6 @@
     <tr><td class="cell-left">${_('tt_sachplan_beschreibung')}</td>            <td>${c['attributes'][description_text] or '-'}</td></tr>
     <tr><td class="cell-left">${_('tt_sachplan_planning_von')}</td>            <td>${c['attributes']['validfrom'] or '-'}</td></tr>
     <tr><td class="cell-left">${_('tt_sachplan_planning_bis')}</td>            <td>${c['attributes']['validuntil'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('tt_sachplan_weitereinfo')}</td>             <td>${c['attributes'][static_url] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('tt_sachplan_objektblatt')}</td>             <td>${c['attributes']['doc_web'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_(objektblatt)}</td>                           <td><a href="${c['attributes']['doc_web'] or '-'}" target="_balnk">${c['attributes']['doc_title'] or '-'}</a></td>
```

Comment:
I would put the ${c['attributes']['doc_web'] into a general if condition, as href="-" leads nowhere.